test/framework/resources/templates/reserve-capacity.yaml.tpl

```diff
@@ -18,6 +18,16 @@ spec:
         app: kubernetes
         role: reserve-capacity
     spec:
+      affinity:
+        podAntiAffinity:
```

Comment:
This deployment doesn't reserve capacity anymore. So conceptually, the name `reserve-capacity` is somehow misleading. 
Should we adapt it to a better fitting name?